pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -948,6 +948,7 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 			Name:        "LINE",
 			Description: "Send notifications to LINE notify",
 			Heading:     "LINE notify settings",
+			Alert:       "Due to the EOL of the LINE Notify API, LINE alerting will stop working after March 31, 2025. Please consider using another contact point.",
```

Comment:
Please check the wording

Also should this be in Japanese?

Reply:
```suggestion
			Alert:       "LINE Notify API の提供終了に伴い、2025 年 3 月 31 日以降は LINE 通知を利用できません。別の Contact Point の利用を検討してください。\n Due to the EOL of the LINE Notify API, LINE alerting will stop working after March 31, 2025. Please consider using another contact point.",
```

そうですね、ほとんど日本のユーザーなので、日本語を追加すると良いと思います。適当に `\n` と書いちゃいましたが、これで正しく改行できるでしょうか？

Reply:
I think including both English and Japanese is the call.  Perhaps we can drop the "Due to ..." line at the start for brevity then include a link to line or our documentation after the end of both (only one link)